src/ert/job_queue/job_queue_node.py

```diff
@@ -110,7 +110,7 @@ def free(self) -> None:
 
     def __str__(self) -> str:
         return (
-            f"JobNode: Name:{self.run_arg.job_name}, Status: {self.queue_status}, "
+            f"ForwardModelStepNode: Name:{self.run_arg.job_name}, Status: {self.queue_status}, "
```

Comment:
I think this JobNode is an actual JobNode, not a ForwardModelStepNode. It is anyways unrelated to this PR..

Reply:
Yes, I agree